src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata minClientVersion="2.8.1">
     <id>Microsoft.DotNet.BuildTools</id>
-    <version>1.0.25-prerelease</version>
+    <version>1.0.26-prerelease</version>
```

Comment:
Don't update the version in the nuspec as this will get updated with a build number after prerelease automatically when we produce a new build tools package.